subcommittees/mentoring/membership-roles.md

```diff
@@ -0,0 +1,68 @@
+# Mentoring Subcommittee Roles and Membership
+
+## Roles
+
+There are multiple roles for participating in mentoring subcommittee activities.  
+
+### Roles for any SWC/DC instructor or instructor trainee
+
+* [Discussion Session Participant](./)
+	- Join a discussion about teaching workshops.  
+* [Discussion Session Host/Co-Host](roles/discussion-hosts.md)
+	- Lead a discussion among fellow instructors about recent or upcoming workshops. 
+* [Mentor](roles/mentors.md)
+	- Work with a small group of new instructors to provide support and guidance.
+* [Mentee](roles/mentees.md)
+	- Work with a mentor and fellow new instructors to prepare for teaching the 
+	Software and Data Carpentry lessons, organizing workshops, and participation in the 
+	Software and Data Carpentry communities.  
+
+Anyone who is a certified instructor, or in the process of becoming certified 
+can participate in our activities in the roles above. 
+
+### Roles for mentoring committee members
+
+* [Chair and Vice-Chair](roles/chair-and-vice.md)
+	- Coordinate mentoring efforts, organize and lead mentoring committee meetings
+* [Secretary](roles/secretary.md)
+	- Take notes in mentoring committee meetings, submit official minutes
+* [Steering Committee Liaison](roles/liaison.md)
+	- Facilitate communication between the steering committee and mentoring committee
+* [Discussion Coordinators](roles/discussion-coordinators.md)
+	- Coordinate volunteer hosts for instructor discussion sessions, contact instructors
+
+These roles are filled by [members](#mentoring-committee-membership) of the 
+mentoring subcommittee, in order to support 
+the operation and activities of the subcommittee.  Current 
+members in these roles are listed in the [README](README.md)
+
+### Choosing Mentoring Committee Member Roles
+
+These roles are filled yearly, in February.  Members can volunteer for any role.  
+Roles are filled in the order listed above, first by consensus (if it exists) and 
+then by a majority vote over a weeklong period via the etherpad if there is not 
+consensus.  Whenever possible roles should
+ be filled by unique people; however, if there are not enough volunteers, 
+ one person may hold up to two positions
+
```

Comment:
How is “consensus” determined?  I'd just start off with a majority vote, but leave the voting medium unspecified.

Also some leading whitespace in the last two lines of the paragraph.